rootfs-builder/README.md

```diff
@@ -158,7 +175,7 @@ After the new directory structure is created:
 After the function `build_rootfs` is called, the script expects the
 rootfs directory to contain `/sbin/init` and `/sbin/kata-agent` binaries.
 
-### Optional - Customise the rootfs
+### Optional - Customize the rootfs
```

Comment:
No comment. Let the Holy Spelling Wars *not* begin... :smile:

Reply:
The spell checker is making the laws now :laughing:

Reply:
Or should that be `lawz`? :smile: